public/components/overview/home.tsx

```diff
@@ -3,159 +3,274 @@
+          pragma: 'no-cache',
+          'sec-fetch-dest': 'empty',
+          'sec-fetch-mode': 'cors',
+          'sec-fetch-site': 'same-origin',
         },
-        {
-          id: alertsURL,
-          title: 'Proactively identify risks with customizable alert triggers.',
-          description: 'with Alerts',
-          exists: alertsPluginExists,
+        method: 'GET',
+        referrerPolicy: 'strict-origin-when-cross-origin',
+        mode: 'cors',
+        credentials: 'include',
+      });
+      const data = await response.json();
+
+      for (const status of data.status.statuses) {
+        if (status.id.includes(alertsPluginID)) {
+          alertsPluginExists = true;
+        }
+        if (status.id.includes(anomalyPluginID)) {
+          anomalyPluginExists = true;
+        }
+      }
+    } catch (error) {
+      console.error('Error checking plugin installation status:', error);
+    }
+
+    cardConfigs
+      .filter((card) => {
+        if (card.id === 'alerts') {
+          return alertsPluginExists;
+        } else if (card.id === 'anomaly') {
+          return anomalyPluginExists;
+        }
+        return true;
+      })
+      .forEach((card: GettingStartedConfig) => {
+        coreRefs.contentManagement?.registerContentProvider({
+          id: card.id,
+          getContent: () => ({
+            id: card.id,
+            kind: 'card',
+            order: card.order,
+            description: card.description,
+            title: card.title,
+            onClick: () => navigateToApp(card.url, '#/'),
+            getFooter: () => {
+              return (
+                // <EuiPanel color="subdued">
+                <EuiText size="s" textAlign="center">
+                  {card.footer}
+                </EuiText>
+                // </EuiPanel>
+              );
+            },
+          }),
+          getTargetArea: () => HOME_CONTENT_AREAS.GET_STARTED,
+        });
+      });
+  };
+
+  const registerDashboard = () => {
+    coreRefs.contentManagement?.registerContentProvider({
+      id: 'dashboard_content',
+      getContent: () => ({
+        id: 'dashboard_content',
+        kind: 'dashboard',
+        order: 1000,
+        input: {
+          kind: 'dynamic',
+          get: () => Promise.resolve(uiSettingsService.get(uiSettingsKey)),
         },
+      }),
+      getTargetArea: () => HOME_CONTENT_AREAS.DASHBOARD,
+    });
+    setIsRegistered(true);
+    const defaultDashboard = uiSettingsService.get(uiSettingsKey);
+    if (dashboards && defaultDashboard && dashboards[defaultDashboard]) {
+      setDashboardTitle(dashboards[defaultDashboard].label);
+      setDashboardId(defaultDashboard);
+      setStartDate(dashboards[defaultDashboard].startDate);
+      setEndDate(dashboards[defaultDashboard].endDate);
+      wrapper.dashboardSelected = true;
+    } else {
+      wrapper.dashboardSelected = false;
+    }
+  };
+
+  useEffect(() => {
+    coreRefs.savedObjectsClient
+      ?.find({
```

Comment:
Instead of repeatedly accessing properties, can we destructure the response at the beginning?
```typescript
useEffect(() => {
  const fetchDashboards = async () => {
    const response = await coreRefs.savedObjectsClient?.find({ type: 'dashboard' });

    const savedDashboards = response?.savedObjects.reduce((acc, { id, attributes }) => {
      const { title, timeFrom, timeTo } = attributes as {
        title: string;
        timeFrom: string;
        timeTo: string;
      };

      acc[id] = {
        value: id,
        label: title,
        startDate: timeFrom,
        endDate: timeTo,
      };
      return acc;
    }, {} as DashboardDictionary);

    setDashboards(savedDashboards || {});

    const defaultDashboard = uiSettingsService.get(uiSettingsKey);

    if (defaultDashboard && savedDashboards?.[defaultDashboard]) {
      const { label, startDate, endDate } = savedDashboards[defaultDashboard];
      setDashboardTitle(label);
      setDashboardId(defaultDashboard);
      setStartDate(startDate);
      setEndDate(endDate);
      wrapper.dashboardSelected = true;
    } else {
      uiSettingsService.set('observability:defaultDashboard', null);
      setDashboardTitle('');
      setDashboardId('');
      wrapper.dashboardSelected = false;
    }
  };

  fetchDashboards();
}, []);
```

Also in the above comment, since we have destructured the `attribute` we do not need the `toString()` for id:
``` typescript
// Original
const dashboardAttributes = savedDashboard.attributes as { title: string; timeFrom: string; timeTo: string };
const id = savedDashboard.id.toString();

// Refactored
const { title, timeFrom, timeTo } = attributes as { title: string; timeFrom: string; timeTo: string };
```

Reply:
Good call out, will look to implement this in a later PR when time allows thorough testing.

Reply:
➕  to Ryan, Let's try to keep the useEffect cleaner, may be move this to function call and function can be declared outside useEffect